### PR TITLE
test haproxy 2.2 and drop 1.9

### PR DESCRIPTION
**Pull request by JohnRoesler**
### Description

1.9 and isn't supported and 2.2 is LTS

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ This file is used to list changes made in each version of the haproxy cookbook.
...
 
+## Unreleased
+
+### Added
+
+- testing for haproxy 2.2
+
+### Removed
+
+- testing for haproxy 1.9
```
Comment (ramereth) [inline CHANGELOG.md:16]: Should 1.8 also be removed?

**Review by ramereth — CHANGES_REQUESTED** [review]
Looks like you need to also fix the suites in `kitchen.yml` to account for this.

File: CHANGELOG.md (hunk 1 above)
Comment (JohnRoesler) [reply CHANGELOG.md:16]: It's LTS through Q4 of 2022, so i figured I'd keep in the mix.

**Review by ramereth — APPROVED** [review]
The suite that fails works for me locally. I'll go ahead and get this released soon.

**Pull request merged**